notebooks/repr_tree.ipynb

```diff
@@ -6,13 +6,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import zarr\n",
-    "\n",
-    "# Python 2/3 trick\n",
-    "try:\n",
-    "    unicode\n",
-    "except NameError:\n",
-    "    unicode = str"
+    "import sys\n",
+    "sys.path.insert(0, '..')\n",
```

Comment:
Why is this needed?

Reply:
So zarr is on the path. How did you do that, did you set PYTHONPATH?